test/parallel/test-repl-pretty-stack.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const repl = require('repl');
+
+
+function run({ command, expected }) {
+  let accum = '';
+
+  const inputStream = new common.ArrayStream();
+  const outputStream = new common.ArrayStream();
+
+  outputStream.write = (data) => accum += data.replace('\r', '');
+
+  const r = repl.start({
+    prompt: '',
+    input: inputStream,
+    output: outputStream,
+    terminal: false,
+    useColors: false
+  });
+
+  r.write(`${command}\n`);
+  assert.strictEqual(accum, expected);
+  r.close();
+}
+
+const tests = [
+  {
+    // test .load for a file that throws
+    command: `.load ${fixtures.path('repl-pretty-stack.js')}`,
+    expected: `Error: Whoops!
+    at repl:9:24
+    at d (repl:12:3)
+    at c (repl:9:3)
+    at b (repl:6:3)
+    at a (repl:3:3)
+`
+  },
+  {
+    command: 'let x y;',
+    expected: `let x y;
+      ^
+
+SyntaxError: Unexpected identifier
+
+`
```

Comment:
I am not a huge fan of the multi line template string and would rather see `\n`

Reply:
Personally, I disagree. Template strings, to my mind, tend to make the test example easier to scan/read quickly.

Reply:
As far as I know this was mainly discouraged so far and the code does not have any multi line template strings. I think it is more difficult to see whitespace with this. I do not want to block the PR with this comment though.

Reply:
Yeah, I'm definitely not a fan of multi-line template strings and we've avoided them consistently in the past. It's definitely something that we can revisit. Ping @nodejs/tsc ... any feelings on this?

Reply:
If it's possible to align the first line with the others (maybe with a tagged literal that removes the first line terminator), then I'm +1.